SFPortfolioApp/SFPortfolioAppUITests/AccessibilityIdentifiers.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  AccessibilityIdentifiers.swift
+//  SFPortfolioAppUITests
+//
+//  Created by Stoyko Kolev on 20.08.21.
+//
+
+import Foundation
+
+struct AccessibilityIdentifiers {
+    
+    static let buttonsCell = "Buttons"
```

Comment:
Should start with lowercase

Reply:
Fixed with creating two separate structs Label and Identifier.